src/components/SectionNotification/SectionNotification.st.css

```diff
@@ -0,0 +1,95 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: value, color, fallback, applyOpacity;
+}
+
+/*Defaults*/
+:vars {
+    BorderRadius: 2px;
+    BorderWidth: 1px;
+
+    MainPadding: 16px;
+    ContentPadding: 8px;
+
+    TextMinWidth: 200px;
+    DefaultTextColor: color(color-5);
+    FontSize: 14px;
+
+    DefaultBackgroundColor: applyOpacity(color(color-5), 0.05);
+    ErrorBackgroundColor: applyOpacity(#F64D43, 0.1);
+}
+
+/*Overrides*/
+:vars {
+    BackgroundColor: --overridable;
+    TextColor: --overridable;
+}
+
+:vars {
+    MainBackgroundColor: color(fallback(value(BackgroundColor), value(DefaultBackgroundColor)));
+    MainTextColor: color(fallback(value(TextColor), value(DefaultTextColor)));
+}
+
+.root {
+    -st-states: error;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    background-color: #FFFFFF;
+    border-radius: value(BorderRadius);
+}
+
+.root:error::main {
+    background-color: value(ErrorBackgroundColor);
+}
+
+.main {
+    display: flex;
+    flex: 1;
+    /* is used to allow controls to wrap on the next line when available width is not enough */
+    flex-wrap: wrap;
+    justify-content: center;
+
+    padding: calc(value(MainPadding) / 2);
+
+    background-color: value(MainBackgroundColor);
+    border: value(BorderWidth) solid applyOpacity(#FFFFFF, 0.4);
+    border-radius: value(BorderRadius);
+
+    color: value(MainTextColor);
+    font-family: "HelveticaNeueW01-55Roma", "Arial", "sans-serif";
+    font-size: value(FontSize);
+}
+
+.contentsWrapper {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    padding: calc(value(MainPadding) / 2);
+}
+
+.buttonsWrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin: calc(value(MainPadding) / 2) calc(value(ContentPadding) / 2);
+}
+
+.buttonsWrapper:empty {
+    display: none;
+}
+
+.text {
+    flex: 1;
+    min-width: value(TextMinWidth);
+    margin: 0 value(ContentPadding);
+}
+
+.text:first-child {
+    margin: 0;
+}
+
+.button  {
+    margin: 0 calc(value(ContentPadding) / 2);
```

Comment:
I think the margin here is incorrect.
In the zeroheight it looks like it's supposed to be `8px`, but here it's `4px`.

Reply:
I know it might be hard to read but I did this intentionally.
To have support for RTL I had to make layout symmetrical, so I've introduced a bit complex margins and paddings so that looks fine on rtl, mobile, with only text, or with only buttons (any number) or all the elements together.
Here is what I mean:
<img width="869" alt="Screen Shot 2020-01-15 at 12 22 31 PM" src="https://user-images.githubusercontent.com/47673738/72426116-d31e8600-3791-11ea-93be-de0a4a868759.png">
<img width="863" alt="Screen Shot 2020-01-15 at 12 22 39 PM" src="https://user-images.githubusercontent.com/47673738/72426123-d6197680-3791-11ea-8a91-94755c87a4b6.png">
<img width="681" alt="Screen Shot 2020-01-15 at 12 22 54 PM" src="https://user-images.githubusercontent.com/47673738/72426130-d87bd080-3791-11ea-8fa2-1130c6c19c76.png">

Reply:
You're right, I missed this!